siliconcompiler/apps/sc_show.py

```
@@ -211,16 +211,20 @@ def __init__(self):
+                               screenshot=show.get("cmdarg", "screenshot"),
+                               tool=show.get("cmdarg", "tool"),
+                               open=show.get("cmdarg", "open"))
 
-    if success and os.path.isfile(success) and show.get("cmdarg", "screenshot"):
-        project.logger.info(f'Screenshot file: {success}')
+        if success and os.path.isfile(success) and show.get("cmdarg", "screenshot"):
+            project.logger.info(f'Screenshot file: {success}')
 
-    return 0
+        return 0
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)